src/components/search/search.stories.tsx

```diff
@@ -40,15 +59,21 @@ export const WithSearchButton: ComponentStory<typeof Search> = () => {
     />
   );
 };
+WithSearchButton.storyName = "With Search Button";
 
-export const DefaultWithColourBackground: ComponentStory<
-  typeof Search
-> = () => {
+export const DefaultWithColourBackground: Story = () => {
   return <Search placeholder="Search..." defaultValue="" />;
 };
-export const WithSearchButtonAndColourBackground: ComponentStory<
-  typeof Search
-> = () => {
+DefaultWithColourBackground.storyName = "Default with Colour Background";
+DefaultWithColourBackground.decorators = [
```

Comment:
Storybook seem to have changed how this works a bit - maybe because of the move away from `<story>` to just having a `<canvas>`. Either way the new changes now do it by adding the decorator to wrap the story in a div with padding and a background colour so the sizing/spacing/colour are all correct and then switching these two stories back to `fullscreen` mode rather than `padding`.

To be honest, I'm not a big fan of these stories and would be better if we just allowed people to switch themes to see how it looks on different backgrounds but that's a future enhancement as part of the story simplification/optimisation work.